shiny/_main.py

```diff
@@ -503,6 +503,32 @@ def try_import_module(module: str) -> Optional[types.ModuleType]:
 }
 
 
+@main.command(
+    help="""Add a test file for a specified app.
+
+Add an empty test file for a specified app. You will be prompted with
+a destination folder. If you don't provide a destination folder, it will be added in the current working
+directory based on the app name.
+
+After creating the shiny app file, you can use `pytest` to run the tests:
+
+        pytest APPDIR/test_APPNAME.py
+"""
+)
+@click.option(
+    "--app",
+    "-a",
+    type=str,
+    help="Please provide the path to the app for which you want to create a test file.",
+)
+def add_tests(
```

Comment:
We should make a `add` method and a `test` sub method. 

Will need to look at https://github.com/posit-dev/py-shinylive/blob/9b489e23e607468344604327bbf88982c2b3775c/shinylive/_main.py#L151-L184 and how it uses the method `@NAME.command` to the sub command.

Reply:
Let's pair through this at the end of the week.